docs/quick_tutorial/authentication/tutorial/security.py

```diff
@@ -1,5 +1,21 @@
-USERS = {'editor': 'editor',
-         'viewer': 'viewer'}
+import os
+import hashlib
+import binascii
+
+SALT_LENGTH = 16
+
+def hash_password_with_salt(password, salt=None):
+    salt = salt or os.urandom(SALT_LENGTH)
+    dk = hashlib.pbkdf2_hmac('sha512', password.encode('utf-8'), salt, 100000)
+    return binascii.hexlify(dk + salt).decode('utf-8')
```

Comment:
@ztane my intent was to show that the values could be changed according to the current recommendation by crypto folks, and suggest not to accept our tutorial as gospel, if that makes sense. IOW, set it and forget it, but set it for what is recommended at that point in time. That might be too much to expect from tutorial users.